packages/compass-home/src/components/home.tsx

```diff
@@ -315,6 +331,75 @@ function Home({
     [setIsWelcomeOpen, appRegistry]
   );
 
+  useEffect(() => {
+    function onAutoupdateStarted() {
+      openToast('update-download', {
+        variant: 'progress',
+        title: 'Compass update is in progress',
+      });
+    }
+    function onAutoupdateFailed() {
+      closeToast('update-download');
+      openToast('update-download', {
```

Comment:
Oh, one small thing that I forgot to mention, it's okay if you want to close it first, but technically `openToast` works closer to `updateToast` (we just copied leafygreen name for the method), so if all your toasts have the same id, they should just update when this is called, no need to close the previous ones first